_includes/catalogpage.html

```diff
@@ -15,7 +15,7 @@
         {% endfor %}
     </ul>
 
-    <p>See more widgets in the <a href="..">Flutter widget catalog</a>.</p>
+    <p>See more widgets in the <a href="/widgets/widgetindex/">Flutter widget catalog</a>.</p>
```

Comment:
Sorry! I'm confused. /widgets is the grid that shows categories

widgetindex is the a-z of all widgets. Do you want "See more widgets" to point to /widgets or widgetindex?